packages/components/src/VirtualizedList/utils/gridrow.test.js

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
+import CellTitle from '../CellTitle';
+import VirtualizedList from '../VirtualizedList.component';
+import {
+	getCellRenderer,
+	getId,
+	getColumnData,
+	getDataKey,
+	getLabel,
+	getRowData,
+	getCellData,
+	extractSpecialFields,
+	renderCell,
+} from './gridrow';
+import { internalIds } from './constants';
+
+const collection = [
+	{
+		id: 0,
+		name: 'Title with icon and actions',
+		description: 'Simple row with icon and actions',
+	},
+	{
+		id: 1,
+		name: 'Title without actions',
+		description: 'Simple row without actions',
+	},
+];
```

Comment:
I've seen that collection definition at least four times. It could be a good idea to extract it.